### ISSUE TITLE
v2.40 submodule error

### ISSUE THREAD

**Issue by marvin-yorke** [ForkIssues/Tracker#2071]
After updating to Version 2.40 some of my submodules display internal errors like ones below, while others seem to work OK. I wonder if there's something wrong with my sumbodule repos or is it indeed a bug in the app itself?

<img width="625" alt="Screenshot 2024-02-21 at 13 04 27" src="https://github.com/fork-dev/Tracker/assets/424279/f74ef585-5ffd-4b22-88ab-4739d83ca292">

<img width="660" alt="Screenshot 2024-02-21 at 13 04 02" src="https://github.com/fork-dev/Tracker/assets/424279/fee76d0f-ed06-413f-a9b3-11f9ed2d2f64">

**Comment by DanPristupov** [comment]
Can you try https://cdn.fork.dev/mac/Fork-2.40.3.dmg?

**Comment by marvin-yorke** [comment]
I've just tried 2.39.1 again (luckily the download link on https://git-fork.com/ is outdated 😄) and it works fine in this regard. 
2.40.3 from above seems to work correctly as well, thanks

**Comment by anbugge-silabs** [comment]
The errors look a little different on my end, but seems to be the same issue. 2.40.3 seems to fix it for me as well, thanks!
![image](https://github.com/fork-dev/Tracker/assets/16835108/58e37cb4-efa7-48e2-bd33-8bf5f29b4baf)

**Comment by StuWebstMW** [comment]
Seeing this error with Fork 2.40.3.

```
🔷> XCGLogger writing log to: file:///Users/****/Library/Logs/Fork.log
🔷[AppDelegate.swift:391] > 
▄████  ████▄ █▄▄▄▄ █  █▀
█▀   ▀ █   █ █  ▄▀ █▄█     Fork 2.40.3
█▀▀    █   █ █▀▀▌  █▀▄     Git: internal
█      ▀████ █  █  █  █    Update Channel: develop
 █             █     █
  ▀           ▀     ▀        https://fork.dev
🔷[WindowManager.swift:228] > Restore workspace 'Work' with 1 windows and 5 tabs
🔷[RefreshRepositoryCommand.swift:198] > Refresh '****************' data. Updated
🔷[RefreshRepositoryCommand.swift:139] > Refresh '****************' status. Updated (1 files)
🔷[GetReferencesGitCommand.swift:63] > bt_get_references (664): 1
⚠️[FileDiffView.swift:915] > Can't read submodule diff content '****/Assets/****'
❌[FileDiffView.swift:706] > bug("refs/remotes/origin/PortObjectiveOrdering != refs/remotes/origin/HEAD")
🔷[RefreshRepositoryCommand.swift:139] > Refresh '****************' status. Updated (1 files)
```
![Screenshot 2024-03-04 at 11 59 47 AM](https://github.com/fork-dev/Tracker/assets/20546310/19cfd3af-b111-47f7-a9d9-1804c26f7715)

**Comment by DanPristupov** [comment]
@StuWebstMW thank you for the bug report. Is this a critical problem for you? Can you wait until the next update?

**Comment by StuWebstMW** [comment]
> @StuWebstMW thank you for the bug report. Is this a critical problem for you? Can you wait until the next update?

Not a critical problem -- no worries waiting. Really enjoying the feature!

**Comment by StuWebstMW** [comment]
Issue appears to be resolved as of v2.41.1. Thanks!

**Comment by DanPristupov** [comment]
@StuWebstMW thank you for the confirmation 👍
